backend/db/models/booking.js

```diff
@@ -31,10 +31,7 @@ module.exports = (sequelize, DataTypes) => {
     },
     startDate: {
       type: DataTypes.DATE,
-      allowNull: false,
-      validate: {
-
-      }
+      allowNull: false
```

Comment:
If you don't have a validate on your model level for startDate, you could remove the empty object. There are some validations you could create though.
Also, forgetting a comma.